Lena14/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Hypertext</title>
+</head>
+
+<body>
+  <section class="google">
+    <h1>Google</h1>
+    <p><a href="https://www.google.com/">The search engine of search engines!</a>
+    </p>
+    <!-- Add a link to google.com here! -->
+  </section>
+
+  <section class="stack-overflow">
+    <h1>Stack Overflow</h1>
+    <p><a href="https://stackoverflow.com/">place for all your programming problems</a></p>
+    <!-- Add a link to stack overflow here! -->
```

Comment:
Good job! But the anchor tags a to be added on this line. They need not be inside the paragraphs. Something like this:
```
<a href="https:/www.stackoverflow.com" target="_blank">
        Link to stack overflow
</a>
```